docs/remote/ssh.md

```diff
@@ -236,7 +236,7 @@ If you would like the local port of the tunnel to be different than the remote s
 
 Right-click the tunnel you want to modify, and select **Change Local Port** in the context menu.
```

Comment:
Should this be "Change Local Address Port" to match the content menu item?

Reply:
I think so, just changed.